automation/01-install-rucio.bash

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env bash
+set -e
+
+# Minikube
+# Possible values for memory: max | format: <number>[<unit>], where unit = b, k, m or g, e.g., 4000mb
+MINIKUBE_MEMORY_RAM=8000mb
+# Possible values for CPUs: max | 1, 2, 3, 4, 5, ...
+MINIKUBE_CPU=4
+
+echo "# --------------------------------------"
+echo "# Check installed packages"
+echo "# --------------------------------------"
+KUBECTL_SUCCESS="The kubectl package is installed."
+KUBECTL_ERROR="The kubectl package is not installed. Please, follow this guide https://kubernetes.io/docs/tasks/tools/install-kubectl/"
```

Comment:
no comma after please :-p

Reply:
and below

Reply:
Fixed in the commit 84bf9a925933eebb4b73482beb79825180033d7d.